server/src/routes/campaign.js

```diff
@@ -1,62 +1,68 @@
-const router = require("express").Router();
-const CampaignController = require("../controllers/campaign");
-const { body } = require("express-validator");
-const validateRequest = require("../middleware/validate-request");
-const requireAuth = require("../middleware/require-auth");
+const router = require('express').Router();
+const CampaignController = require('../controllers/campaign');
+const { body } = require('express-validator');
+const validateRequest = require('../middleware/validate-request');
+const requireAuth = require('../middleware/require-auth');
 
-router.get("/api/campaign", requireAuth, CampaignController.getCampaigns);
+router.get('/api/campaign', requireAuth, CampaignController.getCampaigns);
 
-router.get("/api/campaign/:id", requireAuth, CampaignController.getCampaign);
+router.get('/api/campaign/:id', requireAuth, CampaignController.getCampaign);
```

Comment:
Do you have a different version of prettier btw? A lot of changes just because of the single vs. double quotes